### PR TITLE
Pre-prio: added note about offsite schedule, and specified the channel name

### PR DESCRIPTION
**Pull request by itchingpixels**

### CODE REVIEW

File: meetings/pre-prio.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Mobile pre-prio - Wednesday, 10:45
 
 ASO pre-prio - Wednesday, 11:30
 
+We're planning for 5 days of work (minus conj-time, if we have it on that week), every time, even before the offsite. After the offsite, we're continuing with the previous week's projects, until the first pre-prio, to give time for new projects to materliaze and avoid panic on the first Monday after the gathering.
+
```
Comment (lucjac) [inline meetings/pre-prio.md]: typo materialize

File: meetings/pre-prio.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ Mobile pre-prio - Wednesday, 10:45
 
 ASO pre-prio - Wednesday, 11:30
```
Comment (lucjac) [inline meetings/pre-prio.md]: btw it is unclear to me if the week starts from the pre prio or the prio? E.g. if the prio counts, and the pre prio is in the morning, the wednesday count for the current week not for the next

Comment (itchingpixels) [reply meetings/pre-prio.md]: @lucjac : we haven't specificed this, but let's go with the assumption that it's starting from the prio session

File: meetings/pre-prio.md (hunk 1 above)
Comment (itchingpixels) [reply meetings/pre-prio.md]: thanks, resolved:)

File: meetings/pre-prio.md (hunk 2 above)
Comment (lucjac) [reply meetings/pre-prio.md]: ok!

**Pull request merged**